cpp/ql/src/Best Practices/Likely Errors/CommaBeforeMisleadingIndentation.ql

```diff
@@ -0,0 +1,19 @@
+/**
+ * @name Comma before misleading indentation
+ * @description The expressions before and after the comma operator can be misread because of an unusual difference in start columns.
+ * @kind problem
+ * @id cpp/comma-before-misleading-indentation
+ * @problem.severity recommendation
+ * @tags maintainability
+ *       readability
+ */
+
+import cpp
+
+from CommaExpr ce
+where
+  ce.fromSource() and
+  not exists(MacroInvocation me | ce = me.getAnAffectedElement()) and
```

Comment:
It's indeed a good idea to filter out these results! We actually have a [library predicate](https://github.com/github/codeql/blob/main/cpp/ql/lib/semmle/code/cpp/commons/Exclusions.qll#L126) that handles this for you. I think we should make use of that here.